db/database.go

```diff
@@ -1213,6 +1213,7 @@ func (db *Database) UpdateAllDocChannels(regenerateSequences bool) (int, error)
 						if updatedExpiry != nil {
 							updatedDoc.UpdateExpiry(*updatedExpiry)
 						}
+						doc.SyncData.Crc32UserXattr = base.Crc32cHashString(currentUserXattr)
```

Comment:
Crc32cHashString() is going to return a non-empty string for a nil currentUserXattr.  I think we want to set doc.SyncData.Crc32cUserXattr to empty string if currentUserXattr isn't present.